### ISSUE TITLE
About quasar's config of publicPath

### ISSUE THREAD

**Issue by muei** [quasarframework/quasar#4451]
I config the `quasar.conf.js`, as below:
```
build: {
      scopeHoisting: true,
      vueRouterMode: 'history',
      publicPath: ctx.dev ? '/' : 'http://cdn.xxx.com',
      // vueCompiler: true,
      gzip: true,
      ...
}
```
Because I hope the build files that use as cdn in prod env, but the result is
`index.html`:
```
<!DOCTYPE html>
<html>
<head>
  <title>test</title>
  <link href=/http://cdn.xxx.com/js/4b47640d.7c0909db.js rel=prefetch>
<link href=/http://cdn.xxx.com/css/app.0f5bff89.css rel=preload as=style>
<link href=/http://cdn.xxx.com/js/app.6196755e.js rel=preload as=script>
<link href=/http://cdn.xxx.com/js/runtime.e517ee7b.js rel=preload as=script>
<link href=/http://cdn.xxx.com/js/vendor.10093cfa.js rel=preload as=script>
<link href=/http://cdn.xxx.com/css/app.0f5bff89.css rel=stylesheet>
<base href=/http://cdn.xxx.com/>
<link href=/http://cdn.xxx.com/js/4b47640d.7c0909db.js rel=prefetch>
<link href=/http://cdn.xxx.com/css/app.0f5bff89.css rel=preload as=style>
<link href=/http://cdn.xxx.com/js/app.6196755e.js rel=preload as=script>
<link href=/http://cdn.xxx.com/js/runtime.e517ee7b.js rel=preload as=script>
<link href=/http://cdn.xxx.com/js/vendor.10093cfa.js rel=preload as=script>
<link href=/http://cdn.xxx.com/css/app.0f5bff89.css rel=stylesheet>
<base href=/http://cdn.xxx.com/>
</head>
<body>
  <div id=q-app></div>
  <script type=text/javascript src=/http://cdn.xxx.com/js/app.6196755e.js> </script>
  <script type=text/javascript src=/http://cdn.xxx.com/js/runtime.e517ee7b.js></script>
  <script type=text/javascript src=/http://cdn.xxx.com/js/vendor.10093cfa.js></script>
</body>
</html>
```
Every imported resource use the `publicPath` will has */* prefix, this will not have the correct response

**Comment by rstoenescu** [comment]
Thanks for raising this ticket!
Fix will be available in `@quasar/app` final 1.0.0.

**Comment by muei** [comment]
Otherwise, the `HTML tag` *base* is set as *publicPath*, is that right?
